pkg/vsphere/compute/placement/placement_policy.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2018 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package placement
+
+import (
+	"github.com/vmware/govmomi/object"
+	"github.com/vmware/vic/pkg/trace"
+	"github.com/vmware/vic/pkg/vsphere/performance"
+	"github.com/vmware/vic/pkg/vsphere/vm"
+)
+
+type rankedHost struct {
+	HostReference string
+	*performance.HostMetricsInfo
+	score float64
+}
+
+type rankedHosts []rankedHost
+
+func (r rankedHosts) Len() int           { return len(r) }
```

Comment:
Why are these exported? Doesn't look like it's necessary.

Reply:
I think for `sort.Sort`?